src/ConfirmationRadioButtons/RadioButtonStyled.tsx

```diff
@@ -28,7 +27,7 @@ export const RadioButton: FC<RadioButtonProps> = ({
   onBlur,
   disabled = false,
 }: RadioButtonProps) => (
-  <Box flex alignItems="center">
+  <InputContainer>
```

Comment:
instead of adding a new StyledComponent you can just use `as="label"`

Reply:
@liampuk no what I'm saying is that the S'mores `Confirmation` button should be deprecated in favour of [this](https://marshmallow-insurance.github.io/smores-react/?path=/story/radiogroup--horizontal)

Reply:
Yep that's what I mean, it's maybe worth adding a component that uses `RadioGroup` with Yes/No as that is used quite a lot in agent-portal/fnol?

Reply:
you just need to pass this as options to get a YES/NO component
```
options={[
   { label: 'Yes', value: true },
   { label: 'No', value: false },
]}
 ```
 we do it already in the new direct flow in signup.